StoreApp.BLL/Services/UnitService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using StoreApp.DAL.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StoreApp.BLL.Services
+{
+    public class UnitService : IUnitService
+    {
+        private IUnitOfWork DataBase { get; set; }
+        private IMapper config;
```

Comment:
Look at the style of declaration. I propose to follow one common approach

Reply:
fixed